apps/acceptance/clock/acceptanceStopwatch.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+var Clock = require('./regions/appView');
+var expect = require('chai').expect;
+
+marionette('Clock Acceptance Suite', function(){
+    var client;
+    var clock;
+    client = marionette.client({
+        prefs: {
+            // we need to disable the keyboard
+            // to avoid intermittent failures on
+            // Travis (transitions might take
+            // longer to run and block UI)
+            'dom.mozInputMethod.enabled': false,
+            // Do not require the B2G-desktop app
+            // window to have focus (as per the
+            // system window manager) in order for
+            // it to do focus-related things.
+            'focusmanager.testmode': true
+        },
+        settings: {
+            'ftu.manifestURL': null,
+            'lockscreen.enabled': false
+        }
+    });
+    setup(function() {
+        clock = new Clock(client);
+        clock.launch();
+    });
+    suite('Stopwatch Suite', function() {
+        test('Start Stopwatch and Reset', function() {
+            var stopwatchView = clock.switchView('stopwatch');
+
+            stopwatchView.wait(0.2);
+
+            expect(stopwatchView.getCurrentTime()).to.equal('00:00.00');
+
+            stopwatchView.tapStart();
+
+            stopwatchView.wait(0.2);
+
+            expect(stopwatchView.getCurrentTime()).not.to.equal('00:00.00');
+
+            stopwatchView.tapPause();
+
+            var firstTime = stopwatchView.getCurrentTime();
+
+            stopwatchView.wait(0.2);
+
+            expect(stopwatchView.getCurrentTime()).not.to.equal('00:00.00');
+
+            stopwatchView.tapResume();
+
+            stopwatchView.wait(0.2);
+
+            expect(stopwatchView.getCurrentTime()).not.to.equal('00:00.00');
+
+            expect(stopwatchView.getCurrentTime()).not.to.equal(firstTime);
+
+            stopwatchView.tapPause();
+
+            stopwatchView.wait(0.2);
```

Comment:
Since MJS is straight JS, the actions often happen so fast that the application doesn't have time to react before it's performing the next step. Would you prefer I use waitfors?